src/BlazorUI/Bit.BlazorUI.Extras/Components/RichTextEditor/@types/emitter.d.ts

```
@@ -0,0 +1,33 @@
+        readonly COMPOSITION_START: "composition-start";
+        readonly COMPOSITION_BEFORE_END: "composition-before-end";
+        readonly COMPOSITION_END: "composition-end";
+    };
+    static sources: {
+        readonly API: "api";
+        readonly SILENT: "silent";
+        readonly USER: "user";
+    };
+    protected domListeners: Record<string, {
+        node: Node;
+        handler: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)